screen2.0/src/app/search/ccresearch.tsx

```diff
@@ -333,18 +343,23 @@ export const CcreSearch = (props: { mainQueryParams: MainQueryParams, globals })
           <Box>
             <MainResultsTable
               rows={tableRows}
-              tableTitle={props.mainQueryParams.bed_intersect ?
+              tableTitle={
+                props.mainQueryParams.bed_intersect ?
                 `Intersecting by uploaded .bed file in ${props.mainQueryParams.assembly}${sessionStorage.getItem("warning") === "true" ? " (Partial)" : ""}`
                 :
+                props.mainQueryParams.gene ?
+                `cCREs overlapping ${props.mainQueryParams.gene} - ${props.mainQueryParams.chromosome}:${props.mainQueryParams.start.toLocaleString("en-US")}-${props.mainQueryParams.end.toLocaleString("en-US")}`
```

Comment:
This may conflict with what you're doing if you're working on gene search stuff.

Also this file shows many changes but that's just my IDE doing the auto-format. Only this and minor changes to tabs to make them scrollable